### PR TITLE
V rajdeep/dev 12012022

### PR DESCRIPTION
**Pull request by v-rajdeep**

### CODE REVIEW

File: sdk/communication/communication-calling-server/test/public/callConnection.spec.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ describe("Call Connection Live Test", function() {
       recorder = record(this, environmentSetup);
 
       if (isLiveMode()) {
-        this.skip();
+        // this.skip();
```
Comment (moirf) [inline sdk/communication/communication-calling-server/test/public/callConnection.spec.ts]: do you forget to remove the comment?

File: sdk/communication/communication-calling-server/test/public/callConnection.spec.ts [hunk 2]
```diff
@@ -121,5 +121,60 @@ describe("Call Connection Live Test", function() {
         await callConnection.hangUp();
       }
     });
+
+    it.only("Run hold_resume_audio scenario", async function(this: Context) {
```
Comment (moirf) [inline sdk/communication/communication-calling-server/test/public/callConnection.spec.ts]: hold_resume not changed to remove/add default audio group?

File: sdk/communication/communication-calling-server/test/public/callConnection.spec.ts [hunk 3]
```diff
@@ -33,26 +29,7 @@ describe("Call Connection Live Test", function() {
...
-      };
-      const callConnection = await callingServer.createCallConnection(
-        from_user,
-        [to_user],
-        createCallOptions
-      );
+      const callConnection = await SetupCall();
       try {
         // create PlayAudio option
         const playAudioOptions: PlayAudioOptions = {
```
Comment (moirf) [inline sdk/communication/communication-calling-server/test/public/callConnection.spec.ts:37]: don't put loop true by default, get the variable to set it.

File: sdk/communication/communication-calling-server/test/public/callConnection.spec.ts [hunk 4]
```diff
@@ -69,14 +46,243 @@ describe("Call Connection Live Test", function() {
...
+      const callConnection = await SetupCall();
+      try {
+        const added_participant_id = TestUtils.getFixedUserId(
+          Constants.ParticipantGuid
+        );
+        const participant: CommunicationUserIdentifier = {
+          communicationUserId: added_participant_id
+        };
+        // Add Participant
+        await TestUtils.delayIfLive();
```
Comment (moirf) [inline sdk/communication/communication-calling-server/test/public/callConnection.spec.ts]: Can you check if rename this to waitForOperationCompletion

**Review by moirf — CHANGES_REQUESTED** [review]

File: sdk/communication/communication-calling-server/package.json [hunk 5]
```diff
@@ -79,12 +79,12 @@
     "uuid": "^8.3.0"
   },
```
Comment (v-mausha) [inline sdk/communication/communication-calling-server/package.json:80]: not seeing any new changes, I think no need to check-in this file.

File: sdk/communication/communication-calling-server/test/public/serverCall.spec.ts [hunk 6]
```diff
@@ -68,7 +68,7 @@ describe("Server Call Live Test", function() {
     let toUser: string;
 
     beforeEach(async function(this: Context) {
-      recorder = record(this, environmentSetup);
+        recorder = record(this, environmentSetup);
```
Comment (moirf) [inline sdk/communication/communication-calling-server/test/public/serverCall.spec.ts:71]: does this space intentional?

**Pull request merged**